lib/query.js

```diff
@@ -22,12 +22,15 @@ const idMatch = _.curry((id, obj) => id + '' === obj.id + '');
 const dateIsUnix = _.compose(_.equals(0), _.modulo(_.__, 1));
 
 
-const momentFromString = (date) => moment(new Date(date));
+// dateInMs :: Int -> Bool
+const dateIsMs = (num) => (parseInt(num) > LARGEST_UNIX_TIMESTAMP)
```

Comment:
Don't ever compare different units without converting them to something common. In the case of a problem that you are trying to solve here, there is no reliable solution.

The problem of passing in an integer which may be milliseconds or seconds probably will be okay with our current system, but it is inherently wrong, buggy and evil. You can put a big, big scary comment on top of this function and a reference comment in every place you use this function. In the comment, include boundaries of when this function works. 

But again, don't ever compare different units. 

Perfectly, we should only ever use seconds as a timestamp.

Reply:
Also, pimp-my-sql is a public library. If somebody sees this function, they will mock us for ever and ever. I wouldn't put my username in git on that code.